test.js

```diff
@@ -131,4 +133,31 @@ describe('anymatch', function() {
       matchers.pop();
     });
   });
+
+  describe('windows paths', function() {
+    anymatch.__set__('platform', 'win32');
+
```

Comment:
ok awesome - yes that's a much better approach than the craziness I came up with